apps/web/pages/resume.tsx

```diff
@@ -0,0 +1,341 @@
+const resume = {
+  name: "Philippe Wronski",
+  headline: "Tech Lead / Lead Dev",
+  yearsExperience: "12 ans d’expérience",
```

Comment:
```suggestion
  yearsOfExperience: 12,
```
Je pense qu'ici il fait gérer un nombre, ce sera plus facile à manipuler si besoin